90metalmdsquash/metal-md-lib.sh

```diff
@@ -387,11 +397,10 @@ pave() {
 
     # Select the span of devices we care about; RAID, and all compatible transports.
     doomed_disks="$(lsblk -l -o SIZE,NAME,TYPE,TRAN | grep -E '(raid|'"$metal_transports"')' | sort -u | awk '{print "/dev/"$2}' | tr '\n' ' ' | sed 's/ *$//')"
-    [ -z "$doomed_disks" ] && echo 0 > /tmp/metalpave.done && return 0
 
-    warn nothing can be done to stop this except one one thing...
-    warn ...power this node off within the next 5 seconds to prevent any and all operations...
-    while [ "${time_to_live:-5}" -gt 0 ]; do
+    warn 'nothing can be done to stop this except one one thing ...'
+    warn '... power this node off within the next 5 seconds to prevent any and all operations ...'
```

Comment:
nit: wouldn't it be better to set $time_to_live before this warning so that `5` can be replaced with `$time_to_live` and match reality (in the case where it has been to something different than the default)?

Reply:
Done.